libopenage/presenter/presenter.cpp

```diff
@@ -93,8 +93,13 @@ std::shared_ptr<qtgui::GuiApplication> Presenter::init_window_system() {
 void Presenter::init_graphics(bool debug) {
 	log::log(INFO << "Presenter: Initializing graphics subsystems...");
 
+	// Start up rendering framework
 	this->gui_app = this->init_window_system();
-	this->window = renderer::Window::create("openage presenter test", 1024, 768, debug);
+
+	// Window and renderer
+	renderer::window_settings settings;
+	settings.debug = debug;
+	this->window = renderer::Window::create("openage presenter test", 1024, 768, settings);
```

Comment:
then the resolution should be part of the settings :)

Reply:
great idea :)

Reply:
done